webcomponents/web-components/Tile/tile.js

```diff
@@ -86,7 +92,7 @@ function getTileContent(configuration) {
 
     if (configuration.isLoading === true) {
         const container = document.createElement('div');
-        container.appendChild(getLoader(configuration.iconColor));
+        container.appendChild(getLoader(configuration.spinnerColor));
```

Comment:
If the _spinnerColor_ is null, the _iconColor_ should be used as fallback value (this way it will not break previous usings of this Web Component)